### ISSUE TITLE
[Accessibility] Non-actionable column headers in details are being read out as disabled by narrator, giving a wrong impression about the entire column

### ISSUE THREAD

**Issue by sorosoro2** [OfficeDev/office-ui-fabric-react#4142]
### Bug Report

- __Package version(s)__: 5.55.1001
- __Browser and OS versions__: Windows 10 Fall Update and Microsoft Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Visual Studio Team Services

#### Describe the issue:

Let us have a DetailsList with a column having columnActionsMode set to disabled
When using Narrator with Edge, on reaching the header of this column (say, the header name is Header 1), the narrator reads "Column Header Header1 disabled" 

#### Actual behavior:

The fact that narrator is reading out the term disabled is giving an impression that the entire column is disabled

#### Expected behavior:

If the column header is actionable, we can specify so. But we should not announce disable if the column header is not actionable

### If applicable, please provide a codepen repro:
I found an existing codepen for grouped DetailsList (link below) from the dzearing's library. Please set columnActionsMode to 0 on any column and observe the issue, that I am referring to. 
https://codepen.io/dzearing/pen/LbORXM

**Comment by oengusmacinog** [comment]
@cschlechty do you have thoughts on a best practice in this situation?

**Comment by cschlechty** [comment]
Yes, let's remove aria-disabled from the header.

**Comment by betrue-final-final** [comment]
I spoke with @dzearing yesterday, and the right method would be to make the interactive headers buttons, and the non-interactive headers just text.

**Comment by cschlechty** [comment]
We effectively already have that. We just errantly set a disabled state.

**Comment by ThomasMichon** [comment]
I'm on this.
